test/client/views/requestDetails.js

```diff
@@ -20,13 +37,31 @@ describe('RequestDetailsView', function () {
 
         context('with a request that has an error', function () {
 
-            it('displays that the request has an error');
+            it('displays that the request has an error', function () {
+
+                var hasErrorStub = sinon.stub(this.model, 'hasError');
+
+                hasErrorStub.returns(true);
+
```

Comment:
So from you [comment](https://github.com/mjezzi/tv/pull/2#discussion_r26975910) I put the stub in the test instead of having it in the `before` block. I am not sure if this is what you meant? It did not feel right having it in the `before` block but it also feels kind of weird having the stub here too. Does this look correct?